submissions/jeffersonMormah/CSS/style.css

```diff
@@ -0,0 +1,201 @@
+
+
+.nav-links{
+    width: 98%;
+    margin: auto;
+    overflow: hidden;
+    display: flex;
+     
+}
+ul {
+    padding: 0;
+    margin: 0;
+     
+  }
+
+ 
+
+header {
+    background:rgb(250,128,114);
+    color: #F5F5DC;
+    padding-top: 10px;
+    min-height: 15%;
+    border-bottom:4px solid #0000FF;
+
+}
+
+header a {
+    color: #F5F5DC;
+    font-size: 17px;
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+    font-weight: bold;
+    margin-left: 320px;
+}
+
+header li {
+    display: inline;
+    padding: 0 30px;
+}
+
+header #nav-head {
+    float: left;
+    padding-bottom: 12px;
+}
+header #nav-head h2,header #nav-head p{
+    margin:0;
+    padding:0;
+    color:white;
+}
+
+header nav{
+    float: right;
+    margin-top: 15px;
+}
+
+.ebekel-section {
```

Comment:
I think you should rename this class to actual point to where it is been used or what is been used for

Reply:
I don't understand ur comment pls.

Reply:
@Jeffbuchi Rename the class to be more explanatory to what it is doing or where it is been used.